src/ApplicationLogs/ApplicationLogs.csproj

```diff
@@ -1,9 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>3.0.0-preview3</Version>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <RootNamespace>Neo.Plugins</RootNamespace>
+    <PackageId>Neo.Plugins.ApplicationLogs</PackageId>
```

Comment:
Do we need to explicitly add a PackageId for each project?

Reply:
Nope. Packages will default to the project name if PackageId is not explicitly specified. I figured the packages needed to use the nuget.org reserved ID prefix "Neo".

Reply:
But not all of them are going to be published to the nuget.

Reply:
Neo-Express has dependencies on RpcClient + RpcServer already and will be taking a dependency on Oracle once it's merged. Those modules need to be published on nuget.org. Standardizing the naming for the remaining packages seems like a reasonable idea